template/source/javascripts/modules/in-page-navigation.js

```diff
@@ -0,0 +1,93 @@
+(function($, Modules) {
+  'use strict';
+
+  Modules.InPageNavigation = function InPageNavigation() {
+    var $tocPane;
+    var $contentPane;
+    var $tocItems;
+
+    var animationFrameRequested = false;
+
+    this.start = function start($element) {
+      if (!Modernizr.history) {
```

Comment:
If we include this guard clause here, I'm pretty sure we'll stop the navigation being highlighted on scroll – is that intended?

I guess you could argue that most browsers that don't have the history API also don't have flexbox, so it won't be too useful, but might be nice to decouple that regardless.